tools/graph-links

```diff
@@ -35,10 +36,23 @@ databaseInterface.printSQL = False
 linkUUIDtoNodeName = {}
 
 excludedNodes = {
-    '61c316a6-0a50-4f65-8767-1f44b1eeb6dd': "default fail procedure for transfers. Too many lines",
-    '7d728c39-395f-4892-8193-92f086c0546f': "default fail procedure for SIPs. Too many lines",
-    '333532b9-b7c2-4478-9415-28a3056d58df': "reject transfer option.",
-    '19c94543-14cb-4158-986b-1d2b55723cd8': "reject sip option."}
+    '61c316a6-0a50-4f65-8767-1f44b1eeb6dd': 'default fail procedure for transfers. Too many lines',
+    '7d728c39-395f-4892-8193-92f086c0546f': 'default fail procedure for SIPs. Too many lines',
+    '333532b9-b7c2-4478-9415-28a3056d58df': 'reject transfer option.',
+    '19c94543-14cb-4158-986b-1d2b55723cd8': 'reject sip option',
+}
+
+COLORS = {
+    'node_chain_link': 'black',
+    'node_chain_link_start': 'darkgoldenrod',
+    'edge_exit_codes_ok': 'green',
+    'edge_exit_codes_err': 'red',
+    'edge_user_selections': 'green',
+    'edge_watched_directories': 'yellow',
```

Comment:
User selection edge is the same color as exit code OK.  Also, the yellow is often hard to read against a white background.  While you're changing things, maybe fix this? Black might also work for exit code OK.